extensions/blocks/podcast-episodes/edit.js

```diff
@@ -3,49 +3,126 @@
-import { Placeholder, withNotices } from '@wordpress/components';
-import { useState } from '@wordpress/element';
+import { BlockControls, BlockIcon, InspectorControls } from '@wordpress/block-editor';
+import ServerSideRender from '@wordpress/server-side-render';
 
 /**
  * Internal dependencies
  */
 import './editor.scss';
-import icon from './icon';
-
-function PodcastEpisodesEdit( {
-	attributes,
-	className,
-	noticeOperations,
-	noticeUI,
-	setAttributes,
-} ) {
-	/**
-	 * Write the block editor UI.
-	 *
-	 * @returns {object} The UI displayed when user edits this block.
-	 */
-	const [ notice, setNotice ] = useState();
-
-	/* Call this function when you want to show an error in the placeholder. */
-	const setErrorNotice = () => {
-		noticeOperations.removeAllNotices();
-		noticeOperations.createErrorNotice( __( 'Put error message here.', 'jetpack' ) );
-	};
-
-	return (
-		<div className={ className }>
-			<Placeholder
-				label={ __( 'Podcast Episodes', 'jetpack' ) }
-				instructions={ __( 'Instructions go here.', 'jetpack' ) }
-				icon={ <BlockIcon icon={ icon } /> }
-				notices={ noticeUI }
-			>
-				{ __( 'User input goes here?', 'jetpack' ) }
-			</Placeholder>
-		</div>
-	);
+import { edit, queueMusic } from './icons/';
+
+const DEFAULT_MIN_ITEMS = 1;
+const DEFAULT_MAX_ITEMS = 10;
+
+class PodcastEpisodesEdit extends Component {
+	constructor() {
+		super( ...arguments );
+		this.toggleAttribute = this.toggleAttribute.bind( this );
+		this.onSubmitURL = this.onSubmitURL.bind( this );
+	}
+
+	toggleAttribute( propName ) {
+		return () => {
+			const value = this.props.attributes[ propName ];
+			const { setAttributes } = this.props;
+
+			setAttributes( { [ propName ]: ! value } );
+		};
+	}
+
+	onSubmitURL( event ) {
+		event.preventDefault();
+	}
+
+	render() {
+		const { url, itemsToShow } = this.props.attributes;
+		const { attributes, setAttributes } = this.props;
+
+		if ( ! url ) {
+			return (
+				<Placeholder
+					icon={ <BlockIcon icon={ queueMusic } /> }
+					label={ __( 'Podcast Episodes', 'jetpack' ) }
+					instructions={ __( 'Paste a link to your Podcast RSS feed.', 'jetpack' ) }
+				>
+					<form onSubmit={ this.onSubmitURL }>
+						<TextControl
+							placeholder={ __( 'Enter URL here…', 'jetpack' ) }
+							value={ url || '' }
+							onChange={ value => setAttributes( { url: value } ) }
+							className={ 'components-placeholder__input' }
+						/>
+						<Button isSecondary type="submit">
+							{ __( 'Embed', 'jetpack' ) }
+						</Button>
+					</form>
+					<div className="components-placeholder__learn-more">
+						<ExternalLink href={ __( 'https://wordpress.org/support/article/embeds/' ) }>
+							{ __( 'Learn more about embeds', 'jetpack' ) }
+						</ExternalLink>
+					</div>
+				</Placeholder>
+			);
+		}
+
+		const toolbarControls = [
+			{
+				icon: edit,
+				title: __( 'Edit Podcast Feed URL', 'jetpack' ),
+				onClick: () => this.setState( { editing: true } ),
```

Comment:
Let's either make this work or remove the `onClick` for now and just keep the button without action.

Reply:
Fixed to make it work again.